andino_navigation/package.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>andino_navigation</name>
+  <version>0.0.1</version>
+  <description>
+    Bring up nav2 package with Andino.
+  </description>
+  <maintainer email="olmerg@ekumenlabs.com">Olmer Garcia</maintainer>
+  <maintainer email="franco.c@ekumenlabs.com">Franco Cipollone</maintainer>
+  <license file="LICENSE">BSD Clause 3</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <build_depend>launch_ros</build_depend>
+
+  <exec_depend>launch_ros</exec_depend>
+  <exec_depend>navigation2</exec_depend>
+  <exec_depend>nav2_bringup</exec_depend>
+  <exec_depend>nav2_common</exec_depend>
+  <exec_depend>turtlebot3_gazebo</exec_depend>
```

Comment:
This package is already being brought by `nav2_bringup` so no need to add it explicitly as you are not using it explicitly.

Reply:
I tested an this pkg is required because the gazebo asset has to be installed

Reply:
That's because you are missing `andino_navigation` as dependency. Not because of this

Reply:
Unresolving this thread

Reply:
In my test this package is required because the [gazebo asset](https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/master/turtlebot3_gazebo/models/turtlebot3_world) do not load automatically if I do not install this package

Reply:
Ok, I see why: `nav2_bringup` is missing to add it as dependency in `humble`: https://github.com/ros-planning/navigation2/blob/humble/nav2_bringup/package.xml

However they fixed it for `irwin`/`main`

It's ok then.